lib/firetail.rb

```diff
@@ -166,13 +171,15 @@ def log(env,
 	},
 	response: {
   	  statusCode: status,
-	  body: body ? body.body : body[0],
+	  #body: body[0] ? body[0] : body.body,
+          body: "test",
```

Comment:
Is this expected here?

Reply:
@rileyfiretail Not expected. I had just added some enhancements.